.github/ISSUE_TEMPLATE/deprecation_template.md

```diff
@@ -0,0 +1,19 @@
+# DEPRECATION 
+
+**Feature:**
+
+**Module:**  *python_filename.py*
+ 
+**package** */directory*
+
+## Reason
+
+*Why should this feature be deprecated?  Possible reasons include being outside of pyDARN's current scope or being a legacy function that is no longer used.*
+
+## Replacement
+
+*Would anything replace this feature?*
+
+## Details
+
+*Please provide any other details about the deprecation process and the future.*
```

Comment:
```suggestion
*Please provide any other details about the deprecation process, such as a timeline for warning the user about the deprecation.*
```

Rewording suggestion to provide a possible clarification.  May or may not be useful.